uid-pages/layout-bonita/src/LayoutBonitaV4.json

```diff
@@ -2569,10 +2569,10 @@
       ],
       "exposed" : false
     },
-    "logoutHidden" : {
+    "isSSOActivated" : {
       "type" : "expression",
       "value" : [
-        "return JSON.parse($data.session.conf).indexOf('logout.link.hidden') === -1;"
+        "return JSON.parse($data.session.conf).indexOf('logout.link.hidden') !== -1;"
```

Comment:
that is right. The previous condition was not good or the varible name should have been "logoutVisible"